src/server/services/CheckRole.js

```diff
@@ -23,3 +23,18 @@ module.exports.isAdmin = function(rawRoles) {
   }
   return true;
 };
+
+module.exports.isInstructorOrAdmin = function(rawRoles) {
+  const roles = rawRoles.map(role =>
+    role.substr(role.lastIndexOf('#') + 1, role.length).toLowerCase()
+  );
+  if (
+    !roles.includes('instructor') &&
+    !roles.includes('teacher') &&
```

Comment:
@maxpluez If there's only 'instructor' to check for, then maybe we should remove checking 'teacher' here so it's more consistent throughout the code